docs/pages/datapoint.js

```diff
@@ -104,26 +111,33 @@ Be cautious of using Datapoint for large volumes of data. In cases where the con
           />
         </MainSection.Subsection>
       </MainSection>
+
       <MainSection
         name="Accessibility"
-        description={`Datapoint’s trend prop requires an \`accessibilityLabel\` to describe the trend icon (e.g., Trending up). `}
+        description={`Datapoint's trend prop requires an \`accessibilityLabel\` to describe the trend icon (e.g., Trending up). `}
```

Comment:
replaced all (I think) "smart quotes" with standard apostrophes

Reply:
Cool for now, but would be interested in discussing our stance on smart quotes. From my view, smart quotes would be ideal for content.

Reply:
I'm curious what the advantages are. In general they tend to lead to bugs: either text that someone pastes into a code editor/REPL, or auto-replaced "quote -> smart quote" when creating or editing docs examples (since Prettier doesn't run on those strings). It seems safer to never use them.